src/egraph/check.rs

```diff
@@ -85,8 +85,8 @@ impl<L: Language, N: Analysis<L>> EGraph<L, N> {
                 // eq.r.m :: slots(i) -> X
                 let tmp = eq.l.m.compose_partial(&eq.r.m.inverse());
                 assert!(tmp.is_perm());
-                assert_eq!(c.slots, tmp.keys());
-                assert_eq!(c.slots, tmp.values());
+                assert_eq!(_c.slots, tmp.keys());
+                assert_eq!(_c.slots, tmp.values());
```

Comment:
Feels a bit weird that it's called `_c` even though it *is* still used. I guess that's because of a feature flag.
Doesn't really matter, but you could use `#[allow(unused)]` probably in front of the loop instead, if this bothers you too.